cmd/jaeger/internal/integration/kafka_test.go

```diff
@@ -20,30 +72,52 @@ func TestKafkaStorage(t *testing.T) {
 	// we can change the config to use topic: "${KAFKA_TOPIC:-jaeger-spans}"
 	// and export a KAFKA_TOPIC var with random topic name in the tests.
 
-	collectorConfig := "../../collector-with-kafka.yaml"
-	ingesterConfig := "../../ingester-remote-storage.yaml"
+	// OTLP config files
+	baseCollectorConfig := "../../collector-with-kafka.yaml"
+	baseIngesterConfig := "../../ingester-remote-storage.yaml"
 
-	collector := &E2EStorageIntegration{
-		SkipStorageCleaner:  true,
-		ConfigFile:          collectorConfig,
-		HealthCheckEndpoint: "http://localhost:8888/metrics",
+	formats := []struct {
+		name     string
+		encoding string
+	}{
+		{"OTLP Proto", "otlp_proto"},
+		// {"OTLP JSON", "otlp_json"},
```

Comment:
problem?

Reply:
The Jaeger thrift and OTLP JSON for some reason fail, from the logs can say that the v2 binary refuses to start. So I'm assuming that there's something off with the configs or maybe the formats aren't supported the way we've been trying to use. I'll see if I can come up with something more accurate.